specs/version-api.yaml

```
@@ -0,0 +1,65 @@
+openapi: 3.0.2
+info:
+  version: 0.1.0
+  title: Yagna Version API
+  description: Yagna Version API
+paths:
+  /version/get:
+    get:
+      operationId: getVersion
+      summary: GetVersion - Fetches the version of Yagna
+      responses:
+        '200':
+          $ref: '#/components/responses/Version'
+        '400':
+          $ref: 'common.yaml#/responses/BadRequest'
+        '401':
+          $ref: 'common.yaml#/responses/Unauthorized'
+        default:
+          $ref: 'common.yaml#/responses/UnexpectedError'
+components:
+  responses:
+    Version:
+      description: Yagna Version Response
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/YagnaVersionResponse'
+  schemas:
+    YagnaVersionInfo:
+      type: object
+      properties:
+        version:
+          type: string
+          example: 0.13.2
+        name:
+          type: string
+          example: v0.13.2
```

Name can have something more than just version
```suggestion
          example: v0.12.1 Aware Thrill
```